recipes-bsp/patch-u-boot-env/files/patch-u-boot-env.service

```diff
@@ -13,7 +13,7 @@ Before=swupdate.service shutdown.target
 
 [Service]
 Type=oneshot
-ExecStart=/usr/share/u-boot-env/patch-u-boot-env.sh sysselect /usr/share/u-boot-env/patch-u-boot-env.config
+ExecStart=sh -c '/usr/share/u-boot-env/patch-u-boot-env.sh watchdog_timeout_ms /usr/share/u-boot-env/patch-u-boot-env.config; fw_setenv watchdog_timeout_ms 60000'
```

Comment:
What is the reason for setting the variable `watchdog_timeout_ms` in the u-boot config script and the patch service?

Reply:
The patch service can also handle current firmware 1.2.1 (for non-secure swupdate) by not only setting a non-zero timeout but also adding the related scripts that is missing in that version.